x/gov/codec.proto

```diff
@@ -4,21 +4,24 @@ package gov;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 import "github.com/iov-one/weave/codec.proto";
+import "github.com/iov-one/weave/orm/codec.proto";
 
 // Electorate defines who may vote in an election. This same group can be used in many elections
 // and is stored for re-use
 message Electorate {
   weave.Metadata metadata = 1;
+  // Document version
+  uint32 version = 2;
```

Comment:
Question. Also for @husio 

Would it make sense to add `version` as an optional field to Metadata?
The purpose of using a struct there, and not just one int for schema, was to allow adding optional "Metadata" to many models without modifying then all.

Then again, maybe better to just leave version as an optional field on the model.

Opinions?

Reply:
Within the metadata would be nice as we would not need to support a setter with the model. On  the negative side it adds a field that is currently unused in most cases. So we should use it to ensure WYSIWYG. We had talked about this briefly in Berlin.

Reply:
An unused or 0 field is serialized to 0 bytes.
So there is no runtime overhead to including it there. Just the possible confusion of having this info.
Then again, unversioned objects will always have Version == 0, versioned objects always Version > 0